getconfig.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  All Rights Reserved.
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#
+#  The contents of this software are proprietary and confidential to RidgeRun,
```

Comment:
Remove this header licensing statement. This is open source code, anyone can use it for whatever they want.

Reply:
Apply this change to all the files